components/automate-ui/src/app/entities/notification_rules/notification_rule.model.ts

```diff
@@ -15,6 +15,17 @@ export interface RuleInterface {
   targetType: ServiceActionType;
 }
 
+export interface NotificationTarget {
+  id?: string;
+  url?: string;
+  secret_id?: string;
+  critical_controls_only?: boolean;
+}
+
+export interface NotificationRuleTarget {
+  id?: string;
+  rule: {};
```

Comment:
Looks like you have made an incremental improvement, creating `NotificationRuleTarget`. But why is `rule` just an Object? Is there a reason you cannot make this more specific?